runelite-client/src/main/java/net/runelite/client/plugins/timetracking/clocks/Timer.java

```diff
@@ -40,11 +41,16 @@
 	// the number of seconds remaining on the timer, as of last updated time
 	private long remaining;
 
+	// whether this timer is in the 'warning' state or not
+	@Getter(AccessLevel.NONE)
+	private boolean warning;
```

Comment:
I don't think it makes sense to save the warning, since it is config based and gets reset regardless by the client, so I would make this transient.
```suggestion
	private transient boolean warning;
```